coalib/output/ConsoleInteraction.py

```diff
@@ -540,11 +543,11 @@ def get_action_info(section, action, failed_actions):
     """
     params = action.non_optional_params
 
-    for param_name in params:
+    for param_name in params:  # pragma: no cover
```

Comment:
I don't know that's what I was thinking. I'll check, maybe by mistake I deleted a test